exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num <0) {
+    return true;
+  }  else {
+    return false;
+  }
+}
```

Comment:
this is good, but you could remove the if/else statement and return it all on one line like so:
`return num < 0 `